// hpack.js/test/fixtures.js
exports.specExamples = [
  {
    id: 'C.3.1',
    huffman: false,
    input: '8286 8441 0f77 7777 2e65 7861 6d70 6c65' +
           '2e63 6f6d',
    output: [
      [ ':method', 'GET' ],
      [ ':scheme', 'http' ],
      [ ':path', '/' ],
      [ ':authority', 'www.example.com' ]
    ],
    table: [
      [ ':authority', 'www.example.com', 57 ]
    ]
  },
  {
    id: 'C.3.2',
    continuation: true,
    huffman: false,
    input: '8286 84be 5808 6e6f 2d63 6163 6865',
    output: [
      [ ':method', 'GET' ],
      [ ':scheme', 'http' ],
      [ ':path', '/' ],
      [ ':authority', 'www.example.com' ],
      [ 'cache-control', 'no-cache' ]
    ],
    table: [
      [ 'cache-control', 'no-cache', 53 ],
      [ ':authority', 'www.example.com', 57 ]
    ]
  },
  {
    id: 'C.3.3',
    continuation: true,
    huffman: false,
    input: '8287 85bf 400a 6375 7374 6f6d 2d6b 6579' +
           '0c63 7573 746f 6d2d 7661 6c75 65',
    output: [
      [ ':method', 'GET' ],
      [ ':scheme', 'https' ],
      [ ':path', '/index.html' ],
      [ ':authority', 'www.example.com' ],
      [ 'custom-key', 'custom-value' ]
    ],
    table: [
      [ 'custom-key', 'custom-value', 54 ],
      [ 'cache-control', 'no-cache', 53 ],
      [ ':authority', 'www.example.com', 57 ]
    ]
  },

  {
    id: 'C.4.1',
    input: '8286 8441 8cf1 e3c2 e5f2 3a6b a0ab 90f4' +
           'ff',
    output: [
      [ ':method', 'GET' ],
      [ ':scheme', 'http' ],
      [ ':path', '/' ],
      [ ':authority', 'www.example.com' ]
    ],
    table:
